cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -175,30 +177,30 @@ def related_records(self, key, value):
     _migration = self.get("_migration", {})
     _related = _migration.get("related", [])
     description = None
+    sequence_order = None
     relation_type = OTHER_RELATION.name
-
+    relation_type_tag = clean_val("x", value, str)
+    if relation_type_tag:
```

Comment:
what is the relation_type is none of the one we support?
Don't you want to have an:
```
else:
    raise
```
?